app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -5,13 +5,13 @@ package com.github.se.cyrcle.model.user
  *
  * @property userId The unique identifier of the user.
  * @property username The username of the user.
- * @property profilePictureUrl The path to the profile picture of the user on the cloud storage.
- *   (Not implemented)
+ * @property profilePictureCloudPath The path to the profile picture of the user on the cloud
+ *   storage. (Not implemented)
  */
 data class UserPublic(
     val userId: String,
     val username: String,
-    val profilePictureUrl: String = "",
+    val profilePictureCloudPath: String = "",
```

Comment:
Non-blocking, but I'm not a big fan of the term "Cloud". I have nothing in mind, maybe `ServerStorage` or something, but I'm not even sure it can be describe shortly otherwise.

Reply:
yeah I was trying to find a name that doesn't depend on the current implementation (that is cloud storage) but also that make it clear that it is not the path of the typical firestore storage  we use in the rest of the project

Reply:
I think that sometimes, naming variables really is one of the hardest things in programming ^^